File: 力扣习题/448找到所有数组中消失的数字/problems.py
from typing import List

class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        al = set(nums)
        res = []
        for i in range(1, len(nums) + 1):
            if i not in al:
                res.append(i)
        return res

File: 力扣习题/448找到所有数组中消失的数字/problems.ts
function findDisappearedNumbers(nums: number[]): number[] {
    const alphabet = new Set<number>();
    for(let i = 0; i < nums.length; i++) {
        alphabet.add(nums[i]);
    }
    const res: number[] = [];
    for (let i = 1; i < nums.length + 1; i++) {
        if(!alphabet.has(i)) {
            res.push(i)
        }
    }
    return res;
};